optaplanner-8-benchmarks/optaplanner-perf-benchmark/src/main/java/org/jboss/qa/brms/performance/construction/vrp/VRPTimeWindowedConstructionBenchmark.java

```diff
@@ -34,7 +36,10 @@ public VRPTimeWindowedConstructionBenchmark() {
         ConstructionHeuristicPhaseConfig constructionHeuristicPhaseConfig =
                 new ConstructionHeuristicPhaseConfig().withConstructionHeuristicType(getConstructionHeuristicType());
         solverConfig.setPhaseConfigList(Collections.singletonList(constructionHeuristicPhaseConfig));
-        solverConfig.getEntityClassList().add(TimeWindowedCustomer.class); // diff between normal VRP and TimeWindowed
+
+        List<Class<?>> classes = new ArrayList<>(solverConfig.getEntityClassList());
```

Comment:
This might be avoided if we use a modifiable collection in the basic configuration here: https://github.com/kiegroup/kie-benchmarks/blob/main/optaplanner-8-benchmarks/optaplanner-perf-framework/src/main/java/org/jboss/qa/brms/performance/examples/vehiclerouting/VehicleRoutingExample.java#L55

The `withEntityClasses()` on that line calls `Arrays.asList()` underneath. This shortcut that converts an array to a list, however, uses a `List` implementation that does not support modifications.